### PR TITLE
Remove useless and/or unused CSS animations

### LINKED ISSUE #14268: Remove useless and unused CSS animations
**Describe the issue**
While working on `DashCard` unit test I discovered that `.Card--active` css class is used to set the 30s long animation to the newly added card. Apart from it being too long, it doesn't have any visual effect:
```css
@keyframes fade-out-yellow {
  from {
    background-color: var(--color-bg-white);
  }
  to {
    background-color: var(--color-bg-white);
  }
}

.Dash--editing .DashCard .Card.Card--recent {
  animation-duration: 30s;
  animation-name: fade-out-yellow;
}
```

**Expected behavior**
The animation should either be faster (and it result in some visual effect) or it should be removed. Since we didn't use any effect for newly added dashboard cards, it should be removed.

_There are a few additional completely unused animations floating around that should be removed as well._

**Screenshots**
image

**Severity**
Not sure to what extent this affects performance but it definitely seems like unnecessary additional load.

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING REVIEW

### What does this PR accomplish?
- Removes useless and/or unused CSS animations
- Closes #14268

### CODE REVIEW

File: frontend/src/metabase/css/components/icons.css [hunk 1]
```diff
@@ -11,28 +11,6 @@
...
-
-.Icon--pulse {
-  border-radius: 99px;
-  box-shadow: 0 0 5px var(--color-shadow);
-  padding: 0.75em;
-  animation-name: icon-pulse;
-  animation-duration: 2s;
-  animation-iteration-count: infinite;
-  animation-timing-function: linear;
-}
```
Comment (nemanjaglumac) [inline frontend/src/metabase/css/components/icons.css:34]: Although we do have `.Icon-pulse` in `frontend/src/metabase/icon_paths.js`, I believe this is something different (note the difference between a single and a double dash). Probably some old version of a spinner?

In any case:
1. I didn't find it used anywhere in the project
2. Looking at its keyframes, it doesn't do anything

File: frontend/src/metabase/css/dashboard.css [hunk 2]
```diff
@@ -118,20 +118,6 @@
...
-@keyframes fade-out-yellow {
-  from {
-    background-color: var(--color-bg-white);
-  }
-  to {
-    background-color: var(--color-bg-white);
-  }
-}
-
-.Dash--editing .DashCard .Card.Card--recent {
```
Comment (nemanjaglumac) [inline frontend/src/metabase/css/dashboard.css:130]: `.Card--recent` wasn't used for any other purpose in the project other than this one. It is set in `frontend/src/metabase/dashboard/components/DashCard.jsx:150`

```javascript
className={cx(
  "Card bordered rounded flex flex-column hover-parent hover--visibility",
  {
    "Card--recent": dashcard.isAdded,
    "Card--slow": isSlow === "usually-slow",
  },
)}
```

Do we plan to use that (modifier) class in the future or is it safe to delete it?

Comment (paulrosenzweig) [reply frontend/src/metabase/css/dashboard.css:130]: I think it's safe to delete.

**Review by paulrosenzweig — APPROVED** [review]
Looks good!

**Pull request merged**